app/src/main/java/org/yaaic/db/Database.java

```diff
@@ -457,17 +464,27 @@ private Server populateServer(Cursor cursor)
      * 
      * @return
      */
-    public Cursor getAutoConnectServers()
+    public HashMap<Integer, Server> getAutoConnectServers()
```

Comment:
This method is not needed anymore, is it? getAutoconnectServersAsArrayList() does not call this anymore.